node/taskfiles/taskfile.local.yml

```diff
@@ -246,5 +246,5 @@ tasks:
       - task: test-chain
       - task: test-wss
       - task: test-websocketfetcher
-      - task: test-zeropglog
+      - task: test-logscribeconsumer
```

Comment:
if this logscribeconsumer test is intended to run from `logscribe.test.yaml`, this line should be removed

Reply:
I'll need this command to run logscribeconsumer tests from the logscribe workflow 😄

Reply:
you're running logscribeconsumer test separately not from test command

Reply:
oh i see, got it 👍

Reply:
please remove this line, unless it'll keep running from node test workflow if that's not your intention

Reply:
ah it didn't get committed since I `git add .`-ed from the node directory 😮‍💨. Will push an update